### ISSUE TITLE
`details` tags in assignments lack ARIA attributes

### ISSUE THREAD

**Issue by torgeirl** [devilry/trix2#132]
The assignment editor support some HTML tags such as `<details>` and its `<summary>` that some assignment courses have used to add hints with. These lack ARIA attributes and don't seem to be accessible to all users. We should probably add a better mechanism for hints, that work for all user agents.

----

Related: some of these hints are styled with `<u>` tags, creating underlined elements that aren't links but act like it by toggling the non-summary text of the hint on/off. We should probably make these work in a better way to eliminate hacky workarounds like that.
